### PR TITLE
[WIP] Use nice url for height/profile

### PR DESCRIPTION
**Pull request by procrastinatio**
For at least one year, we have an nice address set for `service-alti` service:
`service-alti.(dev|int|prod).bgdi.ch` and `alti.geo.admin.ch`

We may use it internally

[Test link](//mf-geoadmin3.int.bgdi.ch/mom_use_service_alti/index.html) (should use //service-alti.int.bgdi.ch)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ TECH_SUFFIX = .bgdi.ch
 API_URL ?= //api3.geo.admin.ch
 API_TECH_URL ?= //mf-chsdi3.
 LAST_API_URL := $(call lastvalue,api-url)
-ALTI_URL ?= //api3.geo.admin.ch
+ALTI_URL ?= //alti.geo.admin.ch
 LAST_ALTI_URL := $(call lastvalue,alti-url) 
 ALTI_TECH_URL ?= //mf-chsdi3.
```
Comment (oterral) [inline Makefile]: Just see it should be service-alti no?

Comment (procrastinatio) [reply Makefile]: `alti.geo.admin.ch` or `service-alti.prod.bgdi.ch`

Comment (oterral) [reply Makefile]: So the ALTI_TECH_URL should be //service-alti.

**Comment by gjn** [comment]
Not a fan.

I think we should use the same url as we officially publish. Otherwise, it's just confusing.

**Comment by oterral** [comment]
Why not publishing the new urls ? In particular if we plan a new version of chsdi, that will make sense the user uses the new url right now.

**Comment by danduk82** [comment]
will stick to api3 addresses

**Pull request closed without merging**